rust/poem/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 poem = "~1.1"
-tokio = { version = "~1.14.0", features = ["rt-multi-thread", "macros"] }
+tokio = { version = "~1.15.0", features = ["rt-multi-thread", "macros"] }
```

Comment:
do we need to pin `tokio` in here @sunli829 ?

I mean should / could the `tokio` version handled by `poem` itself ?